src/pluginTypes.ts

```diff
@@ -105,6 +106,7 @@ export type PluginManifest = {
   components?: PluginComponentMap;
   encounterTabs?: Record<string, LazyComponent<React.FC<EncounterTabProps>>>;
   devices?: readonly PluginDeviceManifest[];
+  navItems?: ({ facilityId }: { facilityId?: string }) => NavigationLink[];
```

Comment:
Why is the facilityId passed to navItems? This should be a static list

Reply:
Just so the plugin can decide upon the link, whether it wants to go to `/link`, `/{facilityid}/link` or `/link/{facilityid}`

Reply:
Can you look a bit more into it, facilityId is getting injected into the navitem link even within the core, it should be possible to do that within the core even for NavItems from plugins

Reply:
@gigincg I implemented a different method. Does that work?